flink-runtime/src/test/java/org/apache/flink/runtime/rest/handler/cluster/JobManagerCustomLogHandlerTest.java

```diff
@@ -92,6 +92,7 @@ private static void createFile(File file, String content) throws IOException {
 
     private static HandlerRequest<EmptyRequestBody, FileMessageParameters> createHandlerRequest(
             String path) throws HandlerRequestException {
+        System.out.println(path);
```

Comment:
I guess this is a leftover

Reply:
uuuh, good catch. And thanks - I removed the line that was accidentally push. 😇